pkg/processor/runtime/dotnetcore/test/_outputter/outputter.cs

```diff
@@ -36,6 +42,11 @@ public object outputter(Context context, Event eventBase)
         var body = eventBase.GetBody();
         switch (body)
         {
+            case "check_context":
+                {                    
+                    string json = JsonConvert.SerializeObject(context, Formatting.Indented);
```

Comment:
duplicate variable declaration "json". I'd suggest moving delcartion up and reuse upon each case.

Reply:
Also, `context` may hold information such as DateTime and etc which would lead to undetermine response. perhaps you may want to serialize `context.UserData` and change the case name from `check_context` to `user_data` to better reflect the change